runners/flink/src/main/java/org/apache/beam/runners/flink/FlinkRunnerRegistrar.java

```diff
@@ -0,0 +1,38 @@
+package org.apache.beam.runners.flink;
```

Comment:
Now that we clarified our contribution guidelines, we can merge your contribution. Thanks!